lib/src/instances/option.dart

```diff
@@ -84,11 +228,103 @@ class Some<T> extends Option<T> {
   }
 
   @override
+  bool contains(Object element) => element == value;
+
+  @override
+  T elementAt(int index) {
+    if (index != 0) {
+      throw new RangeError.index(index, this, "index", null, 1);
+    }
+
+    return value;
+  }
+
+  @override
+  bool every(bool f(T element)) => f(value);
+
+  @override
+  IterableMonad<S> expand<S>(Iterable<S> f(T element)) =>
```

Comment:
I am confused, `f` returns an `Iterable`, so unless it is iterated and each entry converted into an `Option` I see no way to turn it into an `Option`, unless is and `Option<Iterable>` but that does not seem to be very useful.